### PR TITLE
Added company name #1

### PR DESCRIPTION
**Pull request by Medy**
Added company name.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-<h1>Nothing</h1>
+<h1>100Devs</h1>
```
Comment (DBarit) [inline index.html]: Can you please add the # to 100Devs? Ex/ #100Devs

Comment (Medy) [reply index.html]: Okay, will do.

**Review by DBarit — CHANGES_REQUESTED** [review]
Please make the changes as requested in the review above. Thank you!

**Review by DBarit — APPROVED** [review]
Looks great! Thanks for all of your hard work on this! Please merge.

**Pull request merged**